examples/DotNetCore/EventHub/EventHubService.cs

```diff
@@ -0,0 +1,156 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Consumer;
+using Azure.Messaging.EventHubs.Processor;
+using Azure.Storage.Blobs;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Newtonsoft.Json;
+
+namespace SimpleAppConfigEventHub
+{
+    public class EventHubService : IEventHubService
+    {
+        public Settings Settings { get; private set; }
+
+        private IConfiguration Configuration;
+
+        private EventHubConnection _eventHubConnection;
+
+        private EventProcessorClient _processorClient = null;
+
+        private IConfigurationRefresher _refresher = null;
+
+        private IEnumerable<string> _kvSelectors = null;
+
+        public EventHubService(IConfiguration configuration)
+        {
+            var builder = new ConfigurationBuilder();
+
+            _eventHubConnection = configuration.GetSection("EventHubConnection").Get<EventHubConnection>();
+            builder.AddAzureAppConfiguration(options =>
+            {
+                options.Connect(configuration["EventHubConnection:AppConfigConnectionString"])
```

Comment:
The App Config connection string shouldn't be under EventHubConnection section. Actually, we should never show an example that putting any connection strings in a config file. All secrets should be from the environment.